package/MDAnalysis/converters/RDKit.py

```diff
@@ -277,8 +305,8 @@ def convert(self, obj, cache=True, NoImplicit=True, max_iter=200,
 
         Parameters
         -----------
-        obj : :class:`~MDAnalysis.core.groups.AtomGroup` or :class:`~MDAnalysis.core.universe.Universe`
-
+        obj : :class:`~MDAnalysis.core.groups.AtomGroup` or
+            :class:`~MDAnalysis.core.universe.Universe`
```

Comment:
has to be on one line to format correctly, see https://mdanalysis--3044.org.readthedocs.build/en/3044/documentation_pages/converters/RDKit.html#MDAnalysis.converters.RDKit.RDKitConverter.convert

Reply:
It might work if you just say
```
obj : `AtomGroup` or `Universe`
```
Try it and see if it links correctly.

Reply:
The back-tic notation did not automagically generate links so your previous solution was better. I'd revert to it.